### PR TITLE
{lib}[gcccuda/2019b] cuTENSOR v1.2.2.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3300.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/4e8e57f3c52395a3899602958f4e5f1c for a full test report.

File: easybuild/easyconfigs/c/cuTENSOR/cuTENSOR-1.2.2.5-gcccuda-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+homepage = 'https://developer.nvidia.com/cutensor'
+description = """The cuTENSOR Library is a GPU-accelerated tensor linear algebra library providing tensor contraction,
+ reduction and elementwise operations."""
+
+toolchain = {'name': 'gcccuda', 'version': '2019b'}
+
+# download requires registration, see https://developer.nvidia.com/cutensor/downloads
+sources = ['libcutensor-linux-%(arch)s-%(version)s.tar.gz']
+checksums = ['954ee22b80d6b82fd4decd42b7faead86af7c3817653b458620a66174e5b89b6']
```
Comment (branfosj) [inline easybuild/easyconfigs/c/cuTENSOR/cuTENSOR-1.2.2.5-gcccuda-2019b.eb:14]:
```suggestion
checksums = [
    {
        'libcutensor-linux-x86_64-1.2.2.5.tar.gz':
            '954ee22b80d6b82fd4decd42b7faead86af7c3817653b458620a66174e5b89b6',
        'libcutensor-linux-ppc64le-1.2.2.5.tar.gz':
            'd914a721b8a6bbfbf4f2bdea3bb51775e5df39abc383d415b3b06bbde2a47e6e',
    }
]
```

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0306u11a.bear.cluster - Linux RHEL 8.2, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/f4d37f714dd51e027115c6d4113e2c32 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0212u15b.bear.cluster - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz (broadwell), Python 3.6.8
See https://gist.github.com/4a4b8da44d8ca7ce94652aa7ffb5c8f1 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**